src/semver/Version.d

```diff
@@ -154,14 +178,20 @@ struct Version
         auto vers = [
             Version.parse("v2.2.0"),
             Version.parse("v1.3.0"),
+            Version.parse("v1.2.4-rc.4"),
             Version.parse("v1.2.3"),
             Version.parse("v1.2.4"),
+            Version.parse("v1.2.4-alpha1"),
```

Comment:
The dot notation should be also done for alphas (`alpha.1`, although if this is testing that the tool doesn't break with just any arbitrary prerelease string, then good).

Reply:
Well, in this case the intention was to test for arbitrary strings. There is no logic for doing that processing for alpha yet